core/runtime/bindings.rs

```diff
@@ -127,7 +127,7 @@ where
     .get(scope, key.into())
     .unwrap_or_else(|| panic!("{path} exists"))
     .try_into()
-    .unwrap_or_else(|_| panic!("unable to convert"))
+    .unwrap_or_else(|_| panic!("Unable to convert {path} to desired type"))
```

Comment:
This funciton should probably be a macro and accept a token tree to print out the desired type nicely